libraries/service/src/main/java/org/holochain/androidserviceruntime/service/HolochainService.kt

```diff
@@ -88,30 +79,35 @@ class HolochainService : Service() {
         Log.d(logTag, "onStartCommand")
 
         when (intent?.action) {
-            ACTION_START -> {
+            HolochainServiceIntentActions.ACTION_START -> {
                 Log.d(
                     logTag,
                     "onStartCommand ACTION_START",
                 )
 
-                // Enable conductor to autostart on system boot
-                getAutostartConfigManager().enable()
+                // Get config from intent
+                val config = intent.extras?.getParcelable<RuntimeNetworkConfigFfiParcel>("config")
```

Comment:
The runtime network config must be passed as an intent "extra" (i.e. some additional data included with the intent).